packages/react/src/accreditation_api_types.ts

```diff
@@ -0,0 +1,58 @@
+import { EntityKind } from './common_api_types'
+
+type AccreditationStatus = 'current' | 'pending' | 'submitter_pending' | 'third_party_pending' | 'expired' | 'rejected'
+type AccreditationAssertionKind = 'income' | 'net-worth' | 'evaluator-assertion' | 'professional-license'
+
+// https://developer.parallelmarkets.com/docs/server/data-structures#accreditation-document
+type AccreditationDocument = {
+  download_url: string
+  download_url_expires: number
+  type: 'certification-letter'
+}
+
+// https://developer.parallelmarkets.com/docs/server/data-structures#individual-accreditation
+type IndividualAccreditation = {
+  id: string
+  status: AccreditationStatus
+  expires_at: number | null
+  assertion_type: AccreditationAssertionKind
+  created_at: number
+  certified_at: number | null
+  first_name: string
+  last_name: string
+  documents: [AccreditationDocument]
+}
+
+// https://developer.parallelmarkets.com/docs/server/data-structures#business-accreditation
+type BusinessAccreditation = {
+  id: string
+  status: AccreditationStatus
+  expires_at: number | null
+  assertion_type: AccreditationAssertionKind
+  created_at: number
+  certified_at: number | null
+  name: string
+  documents: [AccreditationDocument]
+}
+
+// https://developer.parallelmarkets.com/docs/server/accreditations-api
+
+type AccreditationsResponseBase = {
+  id: string
+  user_id: string
+  type: EntityKind
+  indicated_unaccredited: number | null
+  accreditations: [IndividualAccreditation | BusinessAccreditation]
+}
+
+type AccreditationsResponseIndividual = AccreditationsResponseBase & {
+  type: 'individual'
+  accreditations: [IndividualAccreditation]
+}
+
+type AccreditationsResponseBusiness = AccreditationsResponseBase & {
+  type: 'business'
+  accreditations: [BusinessAccreditation]
+}
```

Comment:
By doing this, `AccreditationsApiResponse` is a "discriminated union". That means the user can doing something like 

```ts
if (response.type === 'individual') {
  ... TS now has a type guarantee that the members of the `response.accreditations` array are of type `IndividualAccreditation`, so I don't have to keep checking on subsequent fields, and I can render the right UI _for sure_
```